### DISCUSSION TITLE
How to change delay sync for global effects

### DISCUSSION THREAD

**Discussion by soymonitus** [SynthstromAudible/DelugeFirmware#281] [Ideas]
I always preferred the 8th dotted delay. Now that this is a reality and is selectable in the list of sync options, it would be great if there is a way to select it when Affect All is engaged, either in a Kit or in Song view.
How would you go with this? Any shorcut that you can think of? Maybe an option in the menu to change the click in the golden encoder, to toggle between changing type (Analog/Digital) or going through the list of sync values (8ths, 16ths, 32ths, .... dotted 8ths...). Maybe press and hold knob and turn (but this is not a UX documented way of doing things and probably not a good idea)? Any other idea? Maybe a menu item to change some of the Affect-Entire parameters that cannot be changed through direct actions with golden knobs (turns and clicks)?

**Comment by soymonitus** [comment]
I have seen that there are two parameters for the Delay which are:
SyncType syncType;
SyncLevel syncLevel;

SyncType is an enum with three values: 
	SYNC_TYPE_EVEN = 0,
	SYNC_TYPE_TRIPLET = 10,
	SYNC_TYPE_DOTTED = 19,
Maybe we can add an option to the menu to replace the golden knob click to change between SyncType (upper knob) and SyncLevel (lower knob), instead of PingPong on/off and Analog/Digital respectively. Will there be any possible shortcut to do this without needing a menu option?

**Comment by m-m-adams** [comment]
Are the shortcuts used at all in song view? shift+mod/delay/reverb etc. could be used to edit global effects on that screen. Doesn't help kit affect all though where that currently edits the last touched row

**Reply by soymonitus** [reply]
The feature I developed got merged so now it is possible to edit the SyntType and SyncLevel. You can choose from the community features menu if the clicks in the golden knobs have the default behavior or if they change the new params. Works wonders also for affect-all kits!

**Reply by m-m-adams** [reply]
That leaves the same problem from the opposite direction. Really it exposes that we need a way to edit all song and kit level parameters

**Comment by weavermedia** [comment]
Completed with PR #282 & #290

@soymonitus Is it possible to change this discussion type to Question and mark it as answered?
